### PR TITLE
add disclaimer

### PR DESCRIPTION
**Pull request by jdelbick**
## Description

We had to make this repository public to be able to release it under `@adobe` public npm registry. It is only meant to be consumed by asset-compute-commons and not directly, so we added a disclaimer at the top.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# ⚠️ DISCLAIMER ⚠️
+## This is an internal project. It is only meant to be used by [asset-compute-commons](https://github.com/adobe/asset-compute-commons/blob/master/package.json#L12)
+
```
Comment (tmathern) [inline README.md:3]:
```suggestion
For custom workers and other components needing to use I/O Events, please use the [aio-lib-events](https://github.com/adobe/aio-lib-events) library.
```

**Review by tmathern — CHANGES_REQUESTED** [review]
Needs to link to https://github.com/adobe/aio-lib-events too.

File: README.md (hunk 1 above)
Comment (alexkli) [reply README.md]: I would not mention custom workers?

Comment (jdelbick) [reply README.md]: OK I will remove that

Comment (jdelbick) [reply README.md]: maybe for app builder apps?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# ⚠️ DISCLAIMER ⚠️
+## This is an internal project. It is only meant to be used by [asset-compute-commons](https://github.com/adobe/asset-compute-commons/blob/master/package.json#L12)
+For custom workers and other components needing to use I/O Events, please use the [aio-lib-events](https://github.com/adobe/aio-lib-events) library.
```
Comment (alexkli) [inline README.md:3]: I would not mention custom workers at all? Might just be confusing as they are using it indirectly today (for the asset compute events).

**Review by alexkli — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jdelbick) [reply README.md]: ACK

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# ⚠️ DISCLAIMER ⚠️
+This is an internal project. It is only meant to be used by [asset-compute-commons](https://github.com/adobe/asset-compute-commons/blob/master/package.json#L12)
```
Comment (tmathern) [inline README.md:2]:
```suggestion
This is an internal project. It is only meant to be used by [asset-compute-commons](https://github.com/adobe/asset-compute-commons/blob/master/package.json#L12).
```

**Review by tmathern — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tmathern) [reply README.md]: ACK

**Review by tmathern — APPROVED** [review]

**Pull request merged**